cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature

```diff
@@ -0,0 +1,64 @@
+Feature: IntegratedPlayer
+
+    @initialization
+    Scenario: Launch FCA for 'IntegratedPlayer'
+        Given the environment has been set up for 'IntegratedPlayer' tests
+        And 3rd party 'certification' app is launched with 'inactive' state
```

Comment:
we need the test case to be separate here. 
Creating player can be a single test case
Then all the player method calls like load, play etc should be separate. If we try to keep all of them together in one single test case, if one validation fails, none of the other methods will be validated.

Reply:
api and behavorial testcases seperated with player creation as a single test